Update azure cli docs (#289)
Azure/azure-kusto-python: README.md

* AAD Username/password - Provide your AAD username and password to Kusto client (**check the notice below**).
* AAD application - Provide app ID and app secret to Kusto client.
* AAD code - Provide only your AAD username, and authenticate yourself using a code, generated by ADAL.

> * AZ CLI - For those already using [azure-cli](https://github.com/Azure/azure-cli), provide access token by running `az account get-access-token`.

Reviewer: ```suggestion
* AZ CLI - For those already using [azure-cli](https://github.com/Azure/azure-cli), provide access token for the logged in user`.
```

Reviewer: The python impl. does not actually invoke az commands.
It inspects the az cache for the refresh token and uses it to generate tokens for other resources

Author: Good catch. Did not mean to imply the python implementation invokes az commands.
